checkov/openapi/checks/resource/v2/GlobalSecurityScopeUndefined.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from typing import Any
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.common.checks.enums import BlockType
+from checkov.openapi.checks.resource.v2.BaseOpenapiCheckV2 import BaseOpenapiCheckV2
+
+
+class GlobalSecurityScopeUndefined(BaseOpenapiCheckV2):
+    def __init__(self) -> None:
+        id = "CKV_OPENAPI_19"
+        name = "Ensure that global security scope is defined in securityDefinitions - version 2.0 files"
+        categories = [CheckCategories.API_SECURITY]
+        supported_resources = ["security"]
+        super().__init__(
+            name=name,
+            id=id,
+            categories=categories,
+            supported_entities=supported_resources,
+            block_type=BlockType.DOCUMENT,
+        )
+
+    def scan_openapi_conf(self, conf: dict[str, Any], entity_type: str) -> tuple[CheckResult, dict[str, Any]]:
+        security_definitions = conf.get('securityDefinitions', {}) or {}
+        security_values = conf.get('security', [{}]) or [{}]
```

Comment:
```suggestion
        security_definitions = conf.get('securityDefinitions', {})
        security_values = conf.get('security', [{}])
```

Reply:
did it a bit differently, but the first default is not needed when using `or`

Reply:
my mistake!